package.json

```diff
@@ -19,9 +19,9 @@
   "devDependencies": {
     "autoprefixer": "^10.4.20",
     "docsy": "github:google/docsy#semver:0.11.0",
-    "hugo-extended": "0.143.1",
+    "hugo-extended": "0.144.2",
```

Comment:
Please do not roll version upgrades together with content changes.  This should be a separate PR.